master/internal/command/command.go

```diff
@@ -481,6 +481,15 @@ func (c *command) Receive(ctx *actor.Context) error {
 		}
 		ctx.Respond(&apiv1.SetTensorboardPriorityResponse{Tensorboard: c.toTensorboard(ctx)})
 
+	case *apiv1.DeleteWorkspaceRequest:
+		if c.Metadata.WorkspaceID == model.AccessScopeID(msg.Id) {
+			ctx.Tell(c.allocation, sproto.AllocationSignalWithReason{
```

Comment:
I'd like to add more tests to make sure errors do get propagated

Reply:
More of a nit now: If the underlying implementation of these is just tells, I think the call to `askChildren` should also be a tell instead of an ask. I'm also not sure that Tells will propagate Errors though, you might have to ask in order to do that.